### PR TITLE
UI - Lien Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#14533 &  /bcgov/entity#14532

*Description of changes:*
* Implements LIEN badge in table when property is present on MH Registrations.
* Implements Quick Search button for MHR Number in Transfer flow when LIEN is present.
* Minor UX updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -45,6 +45,21 @@
...
+      <v-row no-gutters>
+        <v-col cols="2"></v-col>
+        <v-col>
+          <v-chip
+            v-if="!isPpr && !isChild && hasLien(item)"
+            class="badge-lien px-3 ml-1"
+            label x-small
+            color="darkGray"
+            text-color="white"
+            data-test-id="owner-removed-badge"
```
Comment (doug-lovett) [inline ppr-ui/src/components/tables/common/TableRow.vue]: May want to change the id: data-test-id = owner-removed-badge ?

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/common/TableRow.vue]: Good catch, yes, thank you! 
Updated 👍

**Review by doug-lovett — APPROVED** [review]
Looks good, just 1 very minor comment.

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 2]
```diff
@@ -705,6 +721,17 @@ export default defineComponent({
...
+    const hasLien = (item: any): boolean => {
+      // Future state might require type handling
+      return !!item.lienRegistrationType
+    }
+
+    const getMhrDescription = (description: APIMhrDescriptionTypes): string => {
+      // Disabled until requirements/verbiage are determined
+      // if (description === APIMhrDescriptionTypes.CONVERTED) return 'REGISTRATION CONVERSION'
+      return description
+    }
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue]: Just to add a comment here: Current placeholder method to map whatever Design requires for a display name for our Converted Registration Types.

**Review by ChasBoucher — APPROVED** [review]
Looks Good..

File: ppr-ui/src/components/tables/common/TableRow.vue (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/tables/common/TableRow.vue]: *Updated to include some mapping for current display Name.

**Pull request merged**